### PR TITLE
Babylon player construction bonus game rule

### PR DESCRIPTION
**Pull request by Danarca**

### CODE REVIEW

**Comment by Danarca** [comment]
Just need a merge conflict fix :)

File: common/game_rules/00_game_rules.txt [hunk 1]
```diff
@@ -131,42 +107,46 @@ monument_effects = {
...
+
+player_construction_bonus_value = {
+	default = enable_pcb_15
+	disable_pcb = {
+	}
+	enable_pcb_15 = {
+	}
+	enable_pcb_051015 = {
+	}
+	enable_pcb_0247 = {
```
Comment (HiddeLekanne) [inline common/game_rules/00_game_rules.txt]: no acronyms, the characters saved don't make up for the readability you lose.

**Review by HiddeLekanne — CHANGES_REQUESTED** [review]

File: common/game_rules/00_game_rules.txt [hunk 2]
```diff
@@ -131,42 +107,46 @@ monument_effects = {
...
 	}
-	
 	disable_customs_union_diplo_plays = {
 	}
-}
\ No newline at end of file
+}
+
+player_construction_bonus_value = {
+	default = enable_player_construction_bonus_15
```
Comment (HiddeLekanne) [inline common/game_rules/00_game_rules.txt]: and default should be single player based

Comment (Danarca) [reply common/game_rules/00_game_rules.txt]: Like so?

Comment (HiddeLekanne) [reply common/game_rules/00_game_rules.txt]: yes

**Review by HiddeLekanne — APPROVED** [review]

**Pull request merged**